cli/copr_cli/main.py

```diff
@@ -590,6 +590,18 @@ def action_mock_config(self, args):
         build_config.rootdir = "{0}-{1}_{2}".format(ownername.replace("@", "group_"), projectname, args.chroot)
         print(MockProfile(build_config))
 
+    @staticmethod
```

Comment:
We are making an instance of this class in the `main` function, so I think we can use a normal (not static) method. Or is there some reason why we need it to be static?